chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -276,6 +280,22 @@
   )
 end
 
+template "/etc/chef/solr.rb" do
+  if chef_solr_tmpfs
+    chef_solr_data = "/dev/shm/solr_data"
+  else
+    chef_solr_data = "/var/cache/chef/solr/data"
+  end
+  source "chef-solr.rb.erb"
+  owner "root"
+  group "chef"
+  mode "0640"
+  variables(
+    solr_heap: chef_solr_heap,
+    solr_data: chef_solr_data
+  )
```

Comment:
Done, I just need to adapt the service to rebuild the index when the service gets started.